releases/firmware-btc-v9.9.0/assertion.txt

```diff
@@ -0,0 +1,8 @@
+By signing this file, the signer confirms that the firmware binary built from:
+
+git tag firmware-btc-only/v9.9.0
+git commit hash ea520f5156a64d3fc57eee8cffae505e674f646e
```

Comment:
this is what i meant with release tags. if i `git clone -b firmware-btc-only/v9.9.0`, i'll end up at this state in the changelog:

![image](https://user-images.githubusercontent.com/25405/149002598-73322628-50ca-43a0-bd59-2372551b3d82.png)

not ideal imho